config/default.json

```diff
@@ -1,7 +1,7 @@
 {
   "autoApproveTokenTransfers": true,
   "defaultVotingMachine": "AbsoluteVote",
-  "gasLimit_deployment": 5500000,
+  "gasLimit_deployment": 5700000,
```

Comment:
How do you get this gas number ?
How about fetching the gas limit from arc package?

Reply:
The Arc value isn't enough for founders.  This is enough for our currently nine founders on Kovan.

See https://github.com/daostack/arc.js/issues/128

Reply:
(I've got this card as the top priority, under a documentation card)

Reply:
You should take the gas limit from arc (which is enough for 3 founders) and then implement 128.

Reply:
Good to know that the Arc limit accounts for 3 founders.  I suggest that the work to do the runtime computation for `forgeOrg` (#128) be done in a separate PR.